internal/services/v1/permissions.go

```diff
@@ -785,6 +785,13 @@ func (a *loadBulkAdapter) Next(_ context.Context) (*tuple.Relationship, error) {
 	a.current.Subject.ObjectID = a.currentBatch[a.numSent].Subject.Object.ObjectId
 	a.current.Subject.Relation = stringz.DefaultEmpty(a.currentBatch[a.numSent].Subject.OptionalRelation, tuple.Ellipsis)
 
+	if a.currentBatch[a.numSent].OptionalExpiresAt != nil {
```

Comment:
please add V1 API test that asserts expiration is being bulk loaded correctly.

Reply:
I did